Link_PreferredBlocks.bas

```diff
@@ -43,10 +51,25 @@ For proCntr = 0 To nmbrPro
         preferredBlock.ProfessorName = professor.ProfessorName
         
         prefBlocks.Add preferredBlock
+     'preferred course
+                
+    'read preferred blocks
+    For nmbrCourse = 1 To numbrCourse
+        Set preferredCourse = New CPreferredCourse
+
+        
+        preferredCourse.PreferredBlockID = nmbrCourse

```

Comment:
why would a preferred course have a preferred Block?
All attributes for a preferredCourse should be consistent to the properties defined in the CPreferredCourse class